### ISSUE TITLE
Uninstall Tool getting blocked as PUA (false-positive)

### ISSUE THREAD

**Issue by aigeros-innov** [crystalidea/uninstall-tool#23]
There's a problem with Sophos Home Antivirus.

Your Product “Uninstall Tool” is getting blocked or removed as Potentially Unwanted Application.

Reason that I'm posting this as an issue here is because one of the developers should contact Sophos for clarifying this directly. I've done it myself several times but as a simple user I'm not privileged to make such a request to them.

If this would be getting fixed, you will have more potential users. **In fact, everyone who uses your software and also uses Sophos Home is most likely a lost customer because of a false-positive.**

Here the newest e-mail which they sent me:

> Hello
> 
> Thank you for contacting Sophos Home support. 
> 
> We apologize for the inconvenience. Potentially unwanted applications (PUA) are programs that are not malicious but may be unsuitable for use in a business environment, and may create security concerns. 
> 
> If you could like, you may submit a sample for reclassification here:
> 
> https://secure2.sophos.com/en-us/support/submit-a-sample.aspx 
> 
> Please feel free to reach out again if you have any questions or concerns.
> Regards,
> Graham - Sophos Home Support

**Comment by kleuter** [comment]
Unfortunately, it's true. We are aware of the problem, Sophos added the app as PUA because AppEsteem markes our app as [deceptor](https://customer.appesteem.com/deceptors?id=UninstallTool-180403). Though we have fixed it long time ago, Sophos doesn't care and ignores our support e-mails replying with absolutely the same template answer. We made 2 phone calls and the support team was unable to help us because it's beyond their jurisdiction. 

We're currently working with AppEsteem and they promised us to have ut fixed once the app gets their certification.

More [info](https://www.google.be/search?q=appesteem&oq=appesteem&aqs=chrome..69i57j69i60l4j35i39.3071j0j4&sourceid=chrome&ie=UTF-8#lrd=0x54906c331412a443:0xa5ace584793b49df,1,,,) on AppEsteem worth checking.
